.github/worflows/cmake.yml

```diff
@@ -1,7 +1,7 @@
 name: CMake-Manually triggered workflow
 
 on:
-  workflow_dispatch:
+  workflow_dispatch
```

Comment:
I think 3 + 4 need to be one line `on: workflow_dispatch` (yaml is finicky)

Reply:
Alright. Done.